demisto_sdk/commands/content_graph/common.py

```diff
@@ -770,3 +770,59 @@ def update(self, other: "Nodes") -> None:  # type: ignore[override]
         "fcm",
     ],
 }
+
+
+CONTENT_PRIVATE_ITEMS: dict = {
```

Comment:
Let's add a comment that this is only temporary and attach a Jira ticket with the proper solution.